src/rpc.cpp

```diff
@@ -484,7 +527,9 @@ RPCServer::RPCServer() {
         }
 
         JSObjectRef credentials_object = (JSObjectRef)jsc::Function::call(m_context, email_password_method, arg_count, arg_values);
-        return (json){{"result", serialize_json_value(credentials_object)}};
+        
+        auto serialized_result = serialize_json_value(credentials_object);
```

Comment:
consider `auto&` or reverting this change

Reply:
Is this to prevent a (potentially expensive) move?

Reply:
I'll revert, this was added while debugging.

Reply:
ok resolving